java/src/main/java/tech/sourced/enry/GoUtils.java

```diff
@@ -3,13 +3,14 @@
 import com.sun.jna.Memory;
 import com.sun.jna.Pointer;
 import tech.sourced.enry.nativelib.GoSlice;
-import tech.sourced.enry.nativelib.GoString;
+import tech.sourced.enry.nativelib._GoString_;
```

Comment:
Hmm, interesting. I wonder why it was changed.

Reply:
The link to the changes is here https://github.com/src-d/enry/pull/164#issuecomment-431672661

Basically, now there 

```
typedef struct { const char *p; intgo n; } _GoString_;
typedef _GoString_ GoString;
```

and the idea is that one can use `_GoString_` to directly access Go string valuse from C  \wo any coping. There are now helpers in C (`_GoStringLen` and `_GoStringPtr` available only in prolog though) that one can use to directly access content of the string.

This Go-Cgo change resulted in a different code  generated by JNA from C->Java, so our helpers were adapted to use new names.

As noted in 5147de9 - there may be another way of adapting for this change